spec/controllers/requests_controller_spec.rb

```diff
@@ -57,7 +57,7 @@
     describe 'unscannable item' do
       it 'redirects to the new mediated page request form' do
         get :new, params: mediated_page_params
-        expect(response).to redirect_to new_mediated_page_path(mediated_page_params)
+        expect(response).to redirect_to new_aeon_page_path(mediated_page_params)
```

Comment:
I'm surprised this test is changing; does the underlying fixture data need to be updated to avoid trigger the Aeon rules?

Reply:
i think this was an old change before i started trying to shift some of the fixture data (e.g. for Hoover archive); I can probably find a way to alter the fixture to prevent it from changing.